### PR TITLE
Address envelope tests

### PR DESCRIPTION
**Pull request by PatStLouis**
Added some helper functions and realigned the envelope tests

### CODE REVIEW

File: tests/fixtures.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/* eslint-disable max-len */
```
Comment (BigBlueHat) [inline tests/fixtures.js:1]: If you use tick mark (`) characters, eslint should not complain.

**Review by BigBlueHat — CHANGES_REQUESTED** [review]
Some wee nits.

File: tests/fixtures.js (hunk 1 above)
Comment (PatStLouis) [reply tests/fixtures.js:1]: I just tried with tick marks and it still complained, are you okay with merging it as is?

Comment (BigBlueHat) [reply tests/fixtures.js:1]: Yeah. Definitely no need to block on this one.

**Review by BigBlueHat — APPROVED** [review]

**Pull request merged**